var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -235,6 +238,15 @@ def nvariant_bool(feature):
                 '-DMPI_Fortran_COMPILER:PATH=%s' % spec['mpi'].mpifc
             ])
 
+        if '+shared' in spec:
+            cmake_args.extend([
```

Comment:
If you're only adding a single string, you can use `append()` instead of `extend([])`